cubic-server/command_parser/commands/Time.cpp

```diff
@@ -0,0 +1,153 @@
+#include "Time.hpp"
+#include "Player.hpp"
+#include "Server.hpp"
+#include "World.hpp"
+
+using namespace command_parser;
+
+void Time::autocomplete(std::vector<std::string> &args, Player *invoker) const
+{
+    if (invoker)
+        return;
+    else
+        LINFO("autocomplete time");
+}
+
+int setTimeFromArg(std::string value, int multiplier)
+{
+    int time = 0;
+    try {
+        if (value.back() == 'd' || value.back() == 's' || value.back() == 't')
```

Comment:
Uh ?

Reply:
According to https://minecraft.fandom.com/wiki/Commands/time :
![image](https://user-images.githubusercontent.com/62648027/231279591-0e830f8b-3b14-45db-b501-8fe4b9657e86.png)